Makefile

```diff
@@ -226,10 +226,10 @@ clean:
 
 ci: ## Run unit tests and benchmark with different configurations
 ci:
-	@ocaml ./configure.ml && BENCH_NAME=vanilla $(MAKE) -s clean test bench-marshal clean
-	@ocaml ./configure.ml --sosa-num && BENCH_NAME=num $(MAKE) -s clean test bench-marshal clean
-	@ocaml ./configure.ml --sosa-zarith && BENCH_NAME=zarith $(MAKE) -s clean test bench-marshal clean
-	@$(MAKE) -s bench-tabulate
+	@ocaml ./configure.ml && BENCH_NAME=vanilla $(MAKE) -s clean test bench-marshal
```

Comment:
I think the first '''clean''' here is not needed because we already made a previous '''make build''' using vanilla just before (vanilla is default), even the configure itself is in fact not needed: we only need to test+bench-marshal what was previously build.